README.md

```diff
@@ -24,7 +24,7 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       # Drafts your next Release notes as Pull Requests are merged into "master"
-      - uses: release-drafter/release-drafter@v5
+      - uses: dream11/release-drafter@v6
```

Comment:
You may add a section at the top of the README file telling from which open source repo we have forked this. You may mention commit as well. In future we might want to take update, although probability of that will be very less.

Reply:
This can be done in a separate PR.